sources/north-america/us/USDA-NAIP.geojson

```diff
@@ -8,21 +8,16 @@
             "required": false
         },
         "name": "National Agriculture Imagery Program",
-        "type": "wms",
-        "url": "https://gis.apfo.usda.gov/arcgis/services/NAIP/USDA_CONUS_PRIME/ImageServer/WMSServer?LAYERS=0&STYLES=&CRS={proj}&BBOX={bbox}&FORMAT=image/jpeg&WIDTH={width}&HEIGHT={height}&VERSION=1.3.0&SERVICE=WMS&REQUEST=GetMap",
-        "available_projections": [
-            "CRS:84",
-            "EPSG:3857",
-            "EPSG:4326"
-        ],
-        "start_date": "2018",
-        "end_date": "2020",
+        "type": "tms",
+        "url": "https://gis.apfo.usda.gov/arcgis/rest/services/NAIP/USDA_CONUS_PRIME/ImageServer/tile/{z}/{y}/{x}",
```

Comment:
[The build](https://github.com/osmlab/editor-layer-index/actions/runs/4149584253/jobs/7221124403) is failing because `{zoom}` is used for the zoom level:

```suggestion
        "url": "https://gis.apfo.usda.gov/arcgis/rest/services/NAIP/USDA_CONUS_PRIME/ImageServer/tile/{zoom}/{y}/{x}",
```